portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -7,6 +7,9 @@
 import javax.servlet.http.HttpServletResponse;
 
 @WebServlet("/form-handler")
+/**
+   * Function that prints the value entered by the user
+   */
```

Comment:
This comment's not formatted properly.

```suggestion
/**
  * Function that prints the value entered by the user
  */
```

Reply:
Also the comment is not reflective of the class. It seems to be reflective of one of it's methods.

Reply:
[7.2](https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment): the fragment is capitalized and punctuated as if it were a complete sentence.